client/src/containers/sidebar/index.tsx

```diff
@@ -61,8 +61,8 @@ export default function Sidebar({
   const widthClassName = useMemo(() => {
     const sectionMaxWidth: Partial<Record<Section, string>> = {
       'geospatial-data': 'w-[min(35%,_490px)]',
-      practices: 'w-[min(45%,_820px)]',
-      network: 'w-[min(45%,_860px)]',
+      practices: 'w-[max(45%,_820px)]',
+      network: 'w-[max(45%,_860px)]',
```

Comment:
Can you explain why you change this?

The problem with this change is that on smaller screen, the sidebar can represent less than 45% of the screen (Geospatial Data) or way more than that (Network).

![On a smaller screen, the sidebar of the Network module takes nearly all the horizontal space](https://github.com/Orcasa-Platform/orcasa/assets/6073968/b898b570-6479-4dc7-969d-c54d710490d7)

Reply:
The network diagram is not prepared for that narrow size of the screen. For network I think is more important to show the information of the network. If we decide to keep it as max we will need some other design for the diagram

Reply:
@Bluesmile82 I think you can easily address this without changing the size of the sidebar. Look at this patch as an example: [proposal.zip](https://github.com/Orcasa-Platform/orcasa/files/13236501/proposal.zip).

I had to zip it to upload it here.